Terminal.Gui/Core/TextFormatter.cs

```diff
@@ -1141,7 +1141,8 @@ public static ustring RemoveHotKeySpecifier (ustring text, int hotPos, Rune hotK
 		/// <param name="normalColor">The color to use for all text except the hotkey</param>
 		/// <param name="hotColor">The color to use to draw the hotkey</param>
 		/// <param name="containerBounds">Specifies the screen-relative location and maximum container size.</param>
-		public void Draw (Rect bounds, Attribute normalColor, Attribute hotColor, Rect containerBounds = default)
+		/// <param name="fillRemaining">Determines if the bounds width will be used (default)or only the text width will be used.</param>
```

Comment:
You're missing a space after `(default)`.

Reply:
I'll fix it, thanks.

Reply:
Done @tig. Thanks for caught that.